hip-config.cmake.in

```diff
@@ -86,8 +86,10 @@ if(HIP_COMPILER STREQUAL "clang")
           ${HIP_CLANG_INCLUDE_SEARCH_PATHS}
       NO_DEFAULT_PATH)
   find_dependency(AMDDeviceLibs)
-  set(AMDGPU_TARGETS "gfx900;gfx906;gfx908" CACHE STRING "AMD GPU targets to compile for")
-  set(GPU_TARGETS "${AMDGPU_TARGETS}" CACHE STRING "GPU targets to compile for")
+  set(GPU_DEFAULT_TARGETS "gfx900;gfx906;gfx908")
+  set(GPU_SUPPORTED_TARGETS "gfx701;gfx801;gfx802;gfx803;gfx900;gfx906;gfx908;gfx1010;gfx1011;gfx1012;gfx1030;gfx1031")
```

Comment:
This is not a complete list of all supported target names. As some targets have [target features](https://llvm.org/docs/AMDGPUUsage.html#target-features) in its name for xnack, sramecc, etc, like `gfx908:xnack+` or `gfx908:xnack-`.